cmd/explorer/main.go

```diff
@@ -530,7 +530,8 @@ func main() {
 			router.HandleFunc("/tools/broadcast", handlers.BroadcastPost).Methods("POST")
 			router.HandleFunc("/tools/broadcast/status/{jobID}", handlers.BroadcastStatus).Methods("GET")
 
-			router.HandleFunc("/tables/state", handlers.DataTableStateChanges).Methods("POST")
+			router.HandleFunc("/tables/getstate", handlers.GetDataTableStateChanges).Methods("POST")
+			router.HandleFunc("/tables/setstate", handlers.SetDataTableStateChanges).Methods("POST")
```

Comment:
The table whose state is stored should be identified in the URI. This would remove the necessity for the key to be in the body of requests, which allows you to use the `GET` method for get requests and remove the error handling for the key inside the handler for both endpoints.
Also, verbs should be avoided in REST API URIs.
Also also, if a resource can be updated multiple times through the same endpoint `PUT` is a more appropriate method than `POST`. 
```suggestion
			router.HandleFunc("/tables/{tableId}/state", handlers.GetDataTableStateChanges).Methods("GET")
			router.HandleFunc("/tables/{tableId}/state", handlers.SetDataTableStateChanges).Methods("PUT")
```

Other parts of the code have to be changed accordingly.

Reply:
Thank you for the improvements, I will keep them in mind for the future.
Everything implemented.